demos/tests/run_tests.py

```diff
@@ -212,7 +214,7 @@ def option_to_args(key, value):
                         try:
                             start_time = timeit.default_timer()
                             subprocess.check_output(fixed_args + dev_arg + case_args,
-                                stderr=subprocess.STDOUT, universal_newlines=True)
+                                stderr=subprocess.STDOUT, universal_newlines=True, encoding='utf-8')
```

Comment:
FWIW, this should be unnecessary once I fix the CI environment (my MR is on review).

Reply:
I've tried run `machine_translation_demo` ru-to-en test on local win10 machine and it crashed without it.

Reply:
@IRDonch ping

Reply:
I'm OK with this; however, I think you should also add a `PYTHONIOENCODING=UTF-8` environment variable to make sure that the child process actually outputs UTF-8.

Reply:
Where? What do you mean?

Reply:
Right here. :) You can add `env={os.environ, 'PYTHONIOENCODING': 'UTF-8'}`.

Reply:
Are you sure that it is right to set env from script? Looks like side effect

Reply:
that should change environment only for process you launched as far as I understand

Reply:
Done